app/src/renderer/components/common/TmBtnCopy.vue

```diff
@@ -26,6 +25,11 @@ export default {
       required: true
     }
   },
+  data: () => ({
+    copy(value) {
```

Comment:
Nice, we can move this to a helper and test is separately, while importing here and AnchorCopy (and in any other places needed)

Reply:
https://github.com/cosmos/voyager/issues/1825